src/pages/runs/{drpdJson.date}/{drpdJson.label}/summary.tsx

```diff
@@ -0,0 +1,135 @@
+import BossCard from "@/components/BossCard";
+import Layout from "@/components/Layout";
+import StarterCard from "@/components/StarterCard";
+import SummaryNotableCaptures from "@/components/SummaryNotableCaptures";
+import TrainerCard from "@/components/TrainerCard";
+import { graphql, PageProps } from "gatsby";
+import React from "react";
+
+
+const SummaryPage: React.FC<PageProps<Queries.SummaryPageQuery>> = ({
+  data,
+  params
+}) => {
+  const drpdJson = data.drpdJson;
+  const date = params.date;
+
+  return (
+    <Layout date={date}>
+      <div className="container px-8 py-8 space-y-6">
+        <h1 className="text-3xl font-bold">Summary</h1>
+        <div className="space-y-4">
+          <div>
+            <h2 className="text-2xl font-bold mb-4">Starters</h2>
+            <div className="grid grid-cols-1 md:grid-cols-3 gap-4">
+              {drpdJson.starters.map((starter, index) => (
+                <StarterCard key={index} pokemon={starter}></StarterCard>
+              ))}
+            </div>
+          </div>
+        </div>
+        <SummaryNotableCaptures drpdJson={drpdJson} />
+        <div>
+          <h2 className="text-2xl font-bold mb-4">Gym Leaders / Boss</h2>
+          <div className="grid grid-cols-2 md:grid-cols-4 gap-4">
+            <TrainerCard
+              trainerId={drpdJson.waves[19].trainer.id}
+              trainerType={"Gym Leader"}
+              name={drpdJson.waves[19].trainer.type}
+              waveNumber={20}
```

Comment:
I wanted to convert 20,30,40 and 50 into constants, any idea what we could name them because I do think this will be used again  in the near future.

Reply:
Yeah that would be a good idea

maybe we can do something like
```ts
const GYM_FLOORS = [20, 30, 40];
const BOSS_FLOOR = 50;
```
?

Reply:
Yeah I think that works fine, thought I don't like how we have to subtract 1 for the trainer name from drdpjson, kinda weird but that's okay